### ISSUE TITLE
How do I get the Tag name? Seems like it was not supported ..

### ISSUE THREAD

**Issue by CaledoniaProject** [PuerkitoBio/goquery#101]
Hi,

Consider something like this, how do I get the tag name within that `func` ?

```
doc.Find("h1, h2, h3, h4, h5, h6").Each(func(i int, s *goquery.Selection) {
})
```

Can you kindly add support for that?

**Comment by PuerkitoBio** [comment]
Hi,

Yeah I'm working on a way to add support for this, this was also requested in #67 (you can see a workaround there in the meantime). I hope to get this merged this week.

Martin

**Comment by PuerkitoBio** [comment]
Closed by #102 .
